playbooks/roles/ses-osh/templates/ses-config.yml.j2

```diff
@@ -1,28 +1,24 @@
-ses_cluster_configuration:
-  ses_cluster_name: ceph
+---
+ceph_conf:
+  cluster_network: {{ hostvars[groups['ses_nodes'][0]].conf_options.stdout_lines.4 }}
+  fsid: {{ hostvars[groups['ses_nodes'][0]].conf_options.stdout_lines.0 }}
+  mon_host: {{ hostvars[groups['ses_nodes'][0]].conf_options.stdout_lines.2 }}
+  mon_initial_members: {{ hostvars[groups['ses_nodes'][0]].conf_options.stdout_lines.1 }}
+  public_network: {{ hostvars[groups['ses_nodes'][0]].conf_options.stdout_lines.3 }}
+cinder:
+  key: {{ ceph_client_cinder_key }}
```

Comment:
I meant with the comment above you can replace the set_fact with the lookup directly here, and avoid the messing up with the input/set fact :)